src/components/cladeFilterDrawer/index.tsx

```diff
@@ -14,7 +14,21 @@ export const CladeFilterDrawer = () => {
   const dispatch = useDispatch();
 
   return (
-    <div style={{ width: windowWidth * 0.3, margin: "auto", marginTop: 50 }}>
+    <div
+      style={{
+        width: windowWidth * 0.4,
```

Comment:
Disregard if you already looked into it, but you might be [better off doing this with `vw` units](https://developer.mozilla.org/en-US/docs/Web/CSS/Viewport_concepts#css) as opposed to needing the react hook to pull the window size. Generally, direct CSS stuff is faster than needing an intermediate JS interaction, but I might also be missing something.